package.json

```diff
@@ -16,7 +16,7 @@
 		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-php && yarn build-packages && yarn build-jetpack && yarn build-jetpack-backup",
 		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'yarn build-php' 'yarn build-packages' 'yarn build-jetpack-concurrently' 'yarn build-jetpack-backup-concurrently'",
 		"build-jetpack": "cd projects/plugins/jetpack && yarn build",
-		"build-jetpack-backup": "cd projects/plugins/jetpack-backup && yarn build",
+		"build-jetpack-backup": "cd projects/plugins/backup && yarn build",
 		"build-jetpack-backup-concurrently": "cd projects/plugins/jetpack-backup && yarn build-concurrently",
```

Comment:
Please update the path to the Backup plugin here too.

Reply:
Thanks for catching that 👍   1a93f01